pyproject.toml

```diff
@@ -6,24 +6,67 @@ authors = ["Joe Banks <joseph@josephbanks.me>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.9"
-starlette = "^0.20.4"
+python = "^3.12"
+starlette = "^0.37.2"
 nested_dict = "^1.61"
-uvicorn = {extras = ["standard"], version = "^0.18.2"}
-motor = "3.0.0"
-python-dotenv = "^0.20.0"
-pyjwt = "^2.4.0"
-httpx = "^0.23.0"
-gunicorn = "^20.1.0"
-pydantic = "^1.8.2"
-spectree = "^0.10.5"
-deepmerge = "^1.0.1"
-sentry-sdk = "^1.9.3"
+uvicorn = { extras = ["standard"], version = "^0.30.1" }
```

Comment:
The `uvicorn.workers` module was depreacted in uvicorn `0.30.0`.
We currently rely on that module for gunicorn in the dockerfile.

We should either install the module from https://github.com/Kludex/uvicorn-worker, or remove gunicorn entirly and just use the `uvicorn` binary in the dockerfile.

My preference is the latter, since we already use k8s for replication, so having gunicorn too seems unneeded.